src/app/components/hris/cv-document/cv-document.component.ts

```diff
@@ -0,0 +1,165 @@
+import { Component, EventEmitter, Output } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import jsPDF from 'jspdf';
+import { EmployeeCertificates } from 'src/app/models/hris/employee-certificates.interface';
+import { EmployeeData } from 'src/app/models/hris/employee-data.interface';
+import { EmployeeProfile } from 'src/app/models/hris/employee-profile.interface';
+import { WorkExperience } from 'src/app/models/hris/work-experience.interface';
+import { EmployeeCertificatesService } from 'src/app/services/hris/employee/employee-certificate.service';
+import { EmployeeDataService } from 'src/app/services/hris/employee/employee-data.service';
+import { EmployeeProfileService } from 'src/app/services/hris/employee/employee-profile.service';
+import { EmployeeQualificationsService } from 'src/app/services/hris/employee/employee-qualifications.service';
+import { WorkExperienceService } from 'src/app/services/hris/employee/employee-work-experience.service';
+import { AuthAccessService } from 'src/app/services/shared-services/auth-access/auth-access.service';
+import { NavService } from 'src/app/services/shared-services/nav-service/nav.service';
+
+@Component({
+  selector: 'app-cv-document',
+  templateUrl: './cv-document.component.html',
+  styleUrls: ['./cv-document.component.css']
+})
+
+export class CvDocumentComponent {
+
+  employeeId = this.route.snapshot.params['id'];
+  selectedEmployee!: EmployeeProfile;
+  employeeProfile!: EmployeeProfile;
+  employeeWorkExp: WorkExperience[] = [];
+  employeeCertificate: EmployeeCertificates[] = [];
+  loggedInProfile!: EmployeeData;
+  skills: any = [];
+  filteredSkills: any = [];
+  name: string | undefined = '';
+  surname: string | undefined = '';
+  role: string | undefined = '';
+  level: number | undefined = 0;
+  education: string | undefined = '';
+  school: string | undefined = '';
+  nqf: number | undefined = 0;
+  skillSet: any | undefined = '';
```

Comment:
Can we refrain from using any. If it's a type then add the model otherwise give it it's type. Looks like a string? Also does it need the additional undefined type if it's being initialized?